compiler/test/runnable/powinline.d

```diff
@@ -0,0 +1,44 @@
+/*
+REQUIRED_ARGS: -O
```

Comment:
This switch is not necessary, since this optimization is done in the frontend, while that switch only affects the backend. Instead, maybe you can put the `-betterC` switch in there, to ensure the rewritten ^^ expressions don't rely on Phobos / druntime  anymore.

Reply:
Addressed in the last commit, https://github.com/dlang/dmd/pull/21082/commits/72754cdebc284e7d87d8dd693d5b2d41a676fdf7.